components/DetailPageSidebar/detail-page-sidebar.scss

```diff
@@ -0,0 +1,85 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/layout';
+
+@include exports("detail-page-sidebar") {
+  .bx--detail-page-sidebar__container {
+    margin: 0 5% 0 0;
```

Comment:
Could you verify that this shorthand is required? Simply `margin-right` might suffice